static/styles/settings.scss

```diff
@@ -1056,15 +1063,9 @@ input[type=checkbox].inline-block {
     margin-top: 8px;
 }
 
-#alert_words_list .edit-alert-word-buttons,
-#attachments_list .edit-attachment-buttons {
```

Comment:
Does this cause any changes in the upload files area?

Reply:
in the compose box (upload file)?

Reply:
Settings page

Reply:
No it seems..

![Uploading Screenshot from 2020-05-10 00-52-46.png…]()